CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+# v1.3.0
+
+:new: What's new:
+
+- Update aws go sdk v2 to support s3 express buckets (#7083)
+- Support long-running dump and restore (#6975)
+- lakectl: Improved performance for lakectl rm recursive command (#7035)
+- PythonSDK: Support StoredObject 'open' as both context and function(#7038)
```

Comment:
remove

Reply:
wasn't sure because it had the include-changelog tag